test/unit/application.spec.js

```diff
@@ -65,20 +65,6 @@ describe('marionette application', function() {
     });
   });
 
-  describe('when an app has been started, and registering another initializer', function() {
```

Comment:
> We haven't quite agreed on what the api should look like, but it'd be something like this:

The problem with this API is that it conflating an 'application' with a 'state.' We should have a single application, and that application should be stateful. The router's role should be to manage the state of that application. Continuing with this thought, things defined on the Router should be states, and not applications.

In the end, the functionality and interface might be similar – even exactly the same – to the end user. But the story we tell to explain the concepts will be **far** easier to understand, I think. The only thing you tap into when you talk about 'sub applications' is maybe getting people excited to think that they're making more modular, less coupled code, when they're really not.